src/components/Icon/index.ts

```diff
@@ -0,0 +1 @@
+export { Icon } from './Icon';
```

Comment:
Curiosidade: faz sentido ter um _default export_ aqui também?
Como `export default Icon`?

Reply:
Não sei se a tua pergunta foi sobre se devemos ou não exportar o Icon para o usuário 😅 

Se está pedindo para exportar, acredito que `export { Icon } from './Icon'` já faz isso, todos os outros componentes seguem essa estrutura. Se for sobre não exportar, foi acordado que iríamos exportar o Icon também